seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/UserServiceImpl.java

```diff
@@ -78,7 +80,8 @@ public AddUserRes add(AddUserReq addReq) {
         res.setId(userId);
 
         // 3. add to role
-        roleServiceImpl.addUserToRole(userId, addReq.getType().intValue());
+        // TODO waiting for optimization
```

Comment:
Why do this?

Reply:
There is no role configuration in the user management function of the front-end prototype file, I suggest that the back-end first comment this part.